manic/repository_git.py

```diff
@@ -380,8 +380,8 @@ def _check_for_valid_ref(self, ref, remote_name, dirname):
         is_tag = self._ref_is_tag(ref, dirname)
         is_branch = self._ref_is_branch(ref, remote_name, dirname)
         is_hash = self._ref_is_hash(ref, dirname)
-
         is_valid = is_tag or is_branch or is_hash
+        print(f"is_tag {is_tag} is_branch {is_branch} is_valid {is_valid}")
```

Comment:
Did you mean to remove this for the final version?

Reply:
I did, thanks.